diff -urN pcre2/src/pcre2_compile.c newpcre2/src/pcre2_compile.c
--- pcre2/src/pcre2_compile.c	2025-10-05 17:18:08.069181032 +0100
+++ newpcre2/src/pcre2_compile.c	2025-10-05 17:19:35.075684173 +0100
@@ -1399,12 +1399,12 @@
 /* Deal with {,m} or n too big. If we successfully read m there is no need to
 check m >= n because n defaults to zero. */
 
-if (!read_number(&p, ptrend, -1, MAX_REPEAT_COUNT, ERR5, &min, errorcodeptr))
+if (!read_number(&p, ptrend, -1, MAX_REPEAT_COUNT, ERR5, (int *)&min, errorcodeptr))
   {
   if (*errorcodeptr != 0) goto EXIT;    /* n too big */
   p++;  /* Skip comma and subsequent spaces */
   while (p < ptrend && (*p == CHAR_SPACE || *p == CHAR_HT)) p++;
-  if (!read_number(&p, ptrend, -1, MAX_REPEAT_COUNT, ERR5, &max, errorcodeptr))
+  if (!read_number(&p, ptrend, -1, MAX_REPEAT_COUNT, ERR5, (int *)&max, errorcodeptr))
     {
     if (*errorcodeptr != 0) goto EXIT;  /* m too big */
     }
@@ -1423,7 +1423,7 @@
     {
     p++;    /* Skip comma and subsequent spaces */
     while (p < ptrend && (*p == CHAR_SPACE || *p == CHAR_HT)) p++;
-    if (!read_number(&p, ptrend, -1, MAX_REPEAT_COUNT, ERR5, &max, errorcodeptr))
+    if (!read_number(&p, ptrend, -1, MAX_REPEAT_COUNT, ERR5, (int *)&max, errorcodeptr))
       {
       if (*errorcodeptr != 0) goto EXIT;   /* m too big */
       }
diff -urN pcre2/src/RELE_PATCHED newpcre2/src/RELE_PATCHED
--- pcre2/src/RELE_PATCHED	1970-01-01 01:00:00.000000000 +0100
+++ newpcre2/src/RELE_PATCHED	2025-10-05 17:21:23.244395542 +0100
@@ -0,0 +1,4 @@
+This directory structure has been patched to allow a cross-compilation.
+
+It's just a couple of minor changes to pcre2_compile.c
+
