diff -urN orig_re2/re2/prog.cc re2/re2/prog.cc
--- orig_re2/re2/prog.cc	2025-10-06 12:56:17.176727295 +0100
+++ re2/re2/prog.cc	2025-10-06 12:57:45.109586303 +0100
@@ -61,7 +61,7 @@
   empty_ = empty;
 }
 
-void Prog::Inst::InitMatch(int32_t id) {
+void Prog::Inst::InitMatch(int id) {
   ABSL_DCHECK_EQ(out_opcode_, uint32_t{0});
   set_opcode(kInstMatch);
   match_id_ = id;
diff -urN orig_re2/re2/RE2_PATCHED re2/re2/RE2_PATCHED
--- orig_re2/re2/RE2_PATCHED	1970-01-01 01:00:00.000000000 +0100
+++ re2/re2/RE2_PATCHED	2025-10-06 13:34:25.157880443 +0100
@@ -0,0 +1,4 @@
+This directory structure has been patched to allow a cross-compilation
+
+It's just a minor change to prog.cc
+
